feature/course-view/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/courseview/ui/LectureListUi.kt

```diff
@@ -63,9 +67,26 @@ private fun LectureListItem(modifier: Modifier, lecture: Lecture, onClick: () ->
         )
     } else stringResource(id = R.string.course_ui_lecture_item_start_date_not_set)
 
-    ListItem(
-        modifier = modifier.clickable(onClick = onClick),
-        headlineContent = { Text(text = lecture.title) },
-        supportingContent = { Text(text = startTimeText) }
-    )
+    Card(
+        modifier = modifier,
+        onClick = onClick
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(8.dp),
+            verticalArrangement = Arrangement.spacedBy(8.dp)
+        ) {
+            // TODO: add teacher image next to title
```

Comment:
Do not just add TODOs into the code, as we will forget about them. Please either fix them now or create a github issue and link it here in the code comment.